handlers/s3client.go

```diff
@@ -0,0 +1,49 @@
+package handlers
+
+import (
+	"os"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/credentials"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/s3"
+	"github.com/aws/aws-sdk-go/service/s3/s3manager"
+)
+
+type AwsClient struct {
```

Comment:
this seems like an unnecessary abstraction

Reply:
It is just AWS session wrapper. I want to have a separation class to handle creating session to AWS S3. It makes the code cleaner for me